Source/Core/DolphinQt/NetPlay/ClickBlurLabel.h

```diff
@@ -0,0 +1,32 @@
+// Copyright 2025 Dolphin Emulator Project
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#pragma once
+
+#include <QGraphicsBlurEffect>
+#include <QLabel>
+#include <QStackedWidget>
+#include <QWidget>
+
+class ClickBlurLabel final : public QWidget
```

Comment:
```suggestion
class ClickBlurLabel final : public QStackedWidget
```
Deriving from `QStackedWidget` itself can eliminate the `QVBoxLayout` and `m_stack` member.